src/tools.rs

```diff
@@ -29,8 +46,21 @@ mod tests {
             let mut rng = rand::thread_rng();
             rng.fill_bytes(&mut seed);
             let module = translate_to_fuzz(&seed);
-            
+
+            assert!(module.is_valid());
+        }
+    }
+
+    #[test]
+    fn test_translate_to_fuzz_mvp() {
+        let mut seed = vec![0; 1000];
+        for _ in 0..1000 {
+            let mut rng = rand::thread_rng();
+            rng.fill_bytes(&mut seed);
+            let module = translate_to_fuzz(&seed);
```

Comment:
I guess you meant to use `translate_to_fuzz_mvp`, right?

Reply:
Oops, yes -- fixed